src/stories/components/404/not-found-component-404.tsx

```diff
@@ -0,0 +1,308 @@
+    marginTop: '132px',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    marginTop: '128px',
+  },
+});
+
+const ImageContainer = styled.div<{ isLight?: boolean }>(({ isLight }) => ({
+  position: 'relative',
+  boxSizing: 'border-box',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  width: '100%',
+  height: '100%',
+  paddingTop: '110px',
+  marginTop: '90px',
+  maxWidth: '343px',
+  margin: '0 auto',
+  '& > span': {
+    borderRadius: '6px',
+    linearGradient: isLight ? 'none' : '180deg, #001020 0%, #000000 63.95%)',
+    boxShadow: isLight ? ' 0px 20px 40px rgba(219, 227, 237, 0.4), 0px 1px 3px rgba(190, 190, 190, 0.25)' : '0px 20px 40px -40px rgba(7, 22, 40, 0.4), 0px 1px 3px rgba(30, 23, 23, 0.25)'
+  },
+  borderRadius: '20px',
+
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    paddingTop: '128px',
+    maxWidth: '769px',
+    margin: '0 auto',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    paddingTop: '59px',
+    maxWidth: '1130px',
+    margin: '0 auto',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    paddingTop: '59px',
+    maxWidth: '1184px',
+    margin: '0 auto',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    paddingTop: '59px',
+    maxWidth: '1184px',
+    margin: '0 auto',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    paddingTop: '59px',
+    maxWidth: '1412px',
+    margin: '0 auto',
+  },
+}));
+
+const ContainerData = styled.div({
+  zIndex: 1,
+  width: '100%',
+  height: '100%',
+  position: 'relative',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+});
+
+const LogoContainer = styled.div({
+  position: 'relative',
+  display: 'block',
+  width: '279px',
+  height: '164px',
+  marginBottom: '64px',
+  marginLeft: '32px',
+  marginRight: '32px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    marginBottom: '160px',
+    width: '580px',
+    height: '340px',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+});
+
+const ContainerText = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  paddingRight: '32px',
+  paddingLeft: '32px',
+  maxWidth: '343px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
```

Comment:
All the media have the same properties, is it possible to use instead:

```css
[lightTheme.breakpoints.up('table_834')]: {
    maxWidth: '100%',
  },
```

Reply:
you are rights, i will fix it

Reply:
you are rights, i will fix it